docs/modules/release-notes/pages/all-releases/4_15S.adoc

```diff
@@ -0,0 +1,198 @@
+only be performed from the *Device Management* page.
+
+With this release, users with the appropriate permission can restart a device by selecting *Restart* via
+
+[NOTE]
+====
+
+* Users without ADMIN role must be assigned a custom role with the permission `org_setting.reboot_private_device` to use this feature.
+* Non-ADMIN users with the reboot permission may only restart devices they can access via their assigned teams.
+* Only devices with *Available* status or *Retained* by the current user can be restarted from the device details.
+
+====
+
+== Option to enable/disable unlimited retention for devices
+
+This release introduces the option to enable/disable unlimited retention of devices for an org. In *Settings → Other Settings*,
+we added the checkbox: *Allow unlimited retention*. Check or uncheck this box, then choose *Save* to enable/disable unlimited retention of devices.
+
+[NOTE]
+====
+
+* To access the *Other Settings* page, the user must be assigned a role with the `permission org_setting.modify`.
+* The *Allow unlimited retention setting* applies to the whole org. By default, this option is `true`.
+
+====
+
+== Monitor number of device offline errors in the last hour
+
+Device admins can now monitor the number of device offline errors within the last hour. This information is available in
+the *Error Count (1hr)* column under *Settings → Device Management*. This feature provides real-time insight, enabling
+quicker identification and resolution of connectivity issues.
+
+If the number of device offline errors in the past hour exceeds 10, a warning icon will appear next to the error count.
+This icon indicates that there may be a potential hardware issue with the device. Hover over the warning icon to view
+suggested actions for resolving the issue.
+
+[NOTE]
+====
+
+In addition to hardware issues, certain actions can cause the device to go offline and increase the error count. These
+actions include restarting services, unplugging and reconnecting the device, or rebooting the Mac mini/Dell server.
+
+====
+
+== Script-based Automation: Launch an Appium Server session using the Flutter Integration driver (Beta)
+
+Previously, we introduced support for launching an automation session using the Appium Flutter driver. With this released, support for automation sessions using the Appium Flutter Integration driver is available.
+
+Before you can start using the Appium Flutter Integration driver, ensure the following prerequisites are met:
+
+* *Flutter App Preparation*: Pre-process your Flutter app-under-test to enable the Flutter Integration driver extension. This step is crucial for the Appium Flutter Integration driver to interact with your app.
+
+* *Server Selection*: Run your automation test script using our basic Appium 2.0 server. To do this, set `'kobiton:runtime'` to `'appium'`. Please note that the Appium Flutter driver is not supported by Xium.
```

Comment:
is the second sentence supposed to be "Appium Flutter Integration driver"?
Also - perhaps we should say: Please note that the Appium Flutter Integration driver cannot be used with Xium, Kobiton's default mobile automation method.